Parsers/ASimAuditEvent/Parsers/ASimAuditEventCiscoISE.yaml

```diff
@@ -0,0 +1,76 @@
+    let ASIM_GetUsernameType = (username:string) { 
+      case ( 
+          username contains "@" , "UPN"
+          , username contains "\\", "Windows"
+          , isempty(username), ""
+          , "Simple"
+      )
+    };
+    let CiscoISEAuditParser=(disabled:bool=false){
+      Syslog
+      | where not(disabled)
+      | where ProcessName has_any ("CISE", "CSCO")
+      | extend EventOriginalType = extract(@".*\d{10}\s(\d{4,5})\s.*", 1, SyslogMessage)
+      | where EventOriginalType in ("52000","52001","52002","52003","52004","52005","52006","52007","52008","52009","52013","52015","52016","52017","52030","52031","52033","52035","52038","52039","52042","52043","52045","52046","52072","52073","52078","52079","52080","52082","52084","52085","52091","52092","52093","57000","58001","58002","58003","58004","58005","60000","60001","60002","60003","60050","60051","60052","60053","60054","60055","60056","60057","60058","60059","60060","60061","60062","60063","60064","60068","60069","60070","60083","60084","60085","60086","60087","60088","60089","60090","60091","60092","60094","60095","60097","60098","60100","60101","60102","60103","60105","60106","60107","60111","60112","60113","60114","60118","60119","60120","60121","60122","60123","60126","60128","60129","60130","60131","60132","60133","60153","60166","60167","60172","60173","60187","60189","60193","60194","60197","60198","60199","60200","60201","60202","60207","60208","60209","60210","60213","60214","60215","60216","60451","60452","61002","61003","61004","61005","61006","61007","61008","61009","61016","61017","61018","61020","61022","61023","61024","61030","61033","61034","61051","61052","61100","61101","61102","61103","61104","61105","61106","61107","61108","61109","61110","61111","61114","61115","61116","61117","61118","61120","61121","61122","61123","61124","61125","61126","61127","61128","61129","61130","61158","61160","61161","61162","61163","61164","61165","61166","61167","61168","61169","61234","62000","62001","62002","62003","61300","61301","61302","60467","60468","62008","62009","62010","62011","62012","62013","62014","62015","62016","62017")
```

Comment:
We don't need to repeat the 200+ Ids here. We can use the EventName datatable with project EventOriginalType and filter in the vim parser can also be applied in this table itself.
For example, for eventresult == "Success", filter this table itself and the rest of the parser will only get the records which are successful. Same is applicable to other filters too.